_includes/additional-practice.md

```diff
@@ -3,14 +3,18 @@ practice solving problems and puzzles. This list, while not
 comprehensive, is a good starting point (and should keep you occupied
 for quite a while).
 
-- [Ruby Monk][ruby-monk]: Interactive lessons from beginning to intermediate Ruby
 - [Code Wars][code-wars]: A collection of coding challenges, rated by difficulty. This one is my favorite, since once you solve a problem, you can see other programmers' solutions.
+
 - [Project Euler][project-euler]: More coding challenges. The first few are fairly straightforward and can be brute-forced. As you get further into the challenges, you'll have to be more clever about optimizing your algorithms.
+
+- [Code Fights][code-fights]: Fun way to practice and test your programming skills against other programming players and bots. Let the fight begin.
+
 - [Code Eval][code-eval]: More coding challenges, broken into three categories by difficulty. Fairly heavy on file input/output, can be difficult to debug.
+
 - [LeetCode][leetcode]: A platform for preparing for technical coding interviews. The code you submit isn't only evaluated for correctness, but speed as well.
 
-[ruby-monk]: http://rubymonk.com
 [code-wars]: http://codewars.com
 [project-euler]: http://projecteuler.net
+[code-fights]: https://codefights.com/
```

Comment:
This page is deprecated. You can keep the updates, but you should also add Code Fights to the [interview prep page](https://github.com/appacademy-prep-work/appacademy-prep-work.github.io/blob/master/_includes/technical-interview.md).